src/components/ContentHighlights/CurrentContentHighlightHeader.jsx

```diff
@@ -80,6 +83,7 @@ const CurrentContentHighlightHeader = ({ enterpriseId }) => {
       <p>
         {HEADER_TEXT.SUB_TEXT.currentContent}
       </p>
+      {isNewArchivedContent ? <ContentHighlightArchivedAlert /> : null }
```

Comment:
See related comment about conditionally rendering `ContentHighlightArchivedAlert` (as is) vs. controlling whether the `Alert` is visible or not via an `open` prop passed here.

Proposed:

```jsx
<ContentHighlightArchivedAlert open={isNewArchivedContent}>
```